frontend/public/templates/work_packages/query_filters.html

```diff
@@ -100,6 +104,7 @@
 
           <span class="inline-label" ng-switch-when="date_past">
             <input ng-model="filter.textValue"
+                   ng-model-options="{ updateOn: 'default blur', debounce: { 'default': 400, 'blur': 0 } }"
```

Comment:
@ulferts Yeah, it might be more sense. I'd prefer to get rid of the very un-DRY `ng-switch-when` though (waiting for angular/angular.js#3410 to be merged).